homeassistant/components/generic/config_flow.py

```diff
@@ -106,6 +107,12 @@ def build_schema(
                 default=user_input.get(CONF_LIMIT_REFETCH_TO_URL_CHANGE, False),
             )
         ] = bool
+        spec[
```

Comment:
Since this is only for advanced use, we can guard it with `if self.show_advanced_options`

Reply:
Nice, I didn't know about that, I'll make that change now.

Reply:
Me neither! Looks good!